mantle/kola/tests/misc/network.go

```diff
@@ -59,21 +60,30 @@ type listener struct {
 func checkListeners(c cluster.TestCluster, expectedListeners []listener) error {
 	m := c.Machines()[0]
 
-	output := c.MustSSH(m, "sudo netstat -plutn")
+	output := c.MustSSH(m, "sudo ss -plutn")
 
 	processes := strings.Split(string(output), "\n")
 	// verify header is as expected
 	if len(processes) < 2 {
-		c.Fatalf("expected at least two lines of nestat output: %q", output)
+		c.Fatalf("expected at least two lines of ss output: %q", output)
 	}
-	if processes[0] != "Active Internet connections (only servers)" {
-		c.Fatalf("netstat output has changed format: %q", output)
-	}
-	if !regexp.MustCompile(`Proto\s+Recv-Q\s+Send-Q\s+Local Address\s+Foreign Address\s+State\s+PID/Program name`).MatchString(processes[1]) {
-		c.Fatalf("netstat output has changed format: %q", output)
+	// ss output's header sometimes does not have whitespace between "Peer Address:Port" and "Process"
+	headerRegex := `Netid\s+State\s+Recv-Q\s+Send-Q\s+Local Address:Port\s+Peer Address:Port\s*Process`
```

Comment:
I think the `sometimes does not have whitespace between "Peer Address:Port" and "Process"` issue probably only happens when you are on a limited width terminal. I can recreate the issue if I'm running in a small width terminal, but if I pipe into `cat` (`sudo ss -plutn | cat`)  then we never have that problem. Since what we're doing here is piping it and not outputting it to a terminal I think you can safely assume you'll never have that missing whitespace problem here.

Reply:
Ah, that would make sense. I found that the test was failing when I forced it to accept the qemu platform; looks like there are no issues when I test it on gcp, though.

Reply:
odd. so you're saying you were able to get the test itself to fail with this problem when running on the qemu platform? I wouldn't have expected that.

Reply:
I meant, in qemu, if I put at least one whitespace character in the regex, there won't be a match, thus failing the test when I run it. It shows that there is no whitespace between "Peer Address: Port" and "Process" when I run on qemu. 
```
network.go:71: ss output has changed format: "Netid State  Recv-Q Send-Q Local Address:Port Peer Address:PortProcess 
```

Reply:
super weird. I'm able to reproduce that. In that case `c.MustSSH` might be doing something weird. Let's just put back in the tolerance for space or no space and add a comment about it being possible to get no space there.